File: tests/fixtures/languages/sample.go
// Minimal Go fixture for language parse-validation tests.

package demo

import "fmt"

type Greeter struct {
	Label string
}

func (g *Greeter) Greet() string {
	return fmt.Sprintf("Hello, %s", g.Label)
}

func Add(a, b int) int {
	return a + b
}

File: tests/fixtures/languages/sample.ml
(* Minimal OCaml fixture for language parse-validation tests. *)

open Printf

module Greeter = struct
  type t = { label : string }

  let make label = { label }

  let greet g = sprintf "Hello, %s" g.label
end

let add a b = a + b

let () = print_endline (Greeter.greet (Greeter.make "world"))
